tensorflow_serving/g3doc/serving_advanced.md

```diff
@@ -143,7 +143,17 @@ With all these, `ServerCore` internally does the following:
   the `SessionBundleSourceAdapter`.
 
 Whenever a new version is available, this `AspiredVersionsManager` always
-unloads the old version and replaces it with the new one. If you want to
+unloads the old version and replaces it with the new one. The default behavior means that
+there will only be one version of the model served at a given time (and that version will be the 
+latest version). The latest version is determined from the model folder name which should be numeric.
+If you would like to serve all the versions available in `model_base_path`, you can set the following environment 
+variable before compiling Tensorflow Serving from source.
+
+~~~
+export TF_SERVING_MULTIPLE_VERSIONS=1
```

Comment:
The tutorials are not intended to be exhaustive, but merely to get users started with basic functionality. I think expecting folks hunting for advanced features to read the config proto definitions (not the codebase) is reasonable. The config proto definitions are generally well documented, and serve as the primary/only source of documentation about advanced features.